config.go

```diff
@@ -51,54 +51,55 @@ type ConfigSection interface {
+	JIRANotifications         JIRANotificationsConfig   `yaml:"jira_notifications" json:"jira_notifications" bson:"jira_notifications" id:"jira_notifications"`
+	Keys                      map[string]string         `yaml:"keys" bson:"keys" json:"keys"`
+	KeysNew                   util.KeyValuePairSlice    `yaml:"keys_new" bson:"keys_new" json:"keys_new"`
+	LDAPRoleMap               LDAPRoleMap               `yaml:"ldap_role_map" bson:"ldap_role_map" json:"ldap_role_map"`
+	LoggerConfig              LoggerConfig              `yaml:"logger_config" bson:"logger_config" json:"logger_config" id:"logger_config"`
+	LogPath                   string                    `yaml:"log_path" bson:"log_path" json:"log_path"`
+	NewRelic                  NewRelicConfig            `yaml:"newrelic" bson:"newrelic" json:"newrelic" id:"newrelic"`
+	Notify                    NotifyConfig              `yaml:"notify" bson:"notify" json:"notify" id:"notify"`
+	Plugins                   PluginConfig              `yaml:"plugins" bson:"plugins" json:"plugins"`
+	PluginsNew                util.KeyValuePairSlice    `yaml:"plugins_new" bson:"plugins_new" json:"plugins_new"`
+	PprofPort                 string                    `yaml:"pprof_port" bson:"pprof_port" json:"pprof_port"`
+	Providers                 CloudProviders            `yaml:"providers" bson:"providers" json:"providers" id:"providers"`
+	RepoTracker               RepoTrackerConfig         `yaml:"repotracker" bson:"repotracker" json:"repotracker" id:"repotracker"`
+	Scheduler                 SchedulerConfig           `yaml:"scheduler" bson:"scheduler" json:"scheduler" id:"scheduler"`
+	ServiceFlags              ServiceFlags              `bson:"service_flags" json:"service_flags" id:"service_flags" yaml:"service_flags"`
+	SSHKeyDirectory           string                    `yaml:"ssh_key_directory" bson:"ssh_key_directory" json:"ssh_key_directory"`
+	SSHKeyPairs               []SSHKeyPair              `yaml:"ssh_key_pairs" bson:"ssh_key_pairs" json:"ssh_key_pairs"`
+	Slack                     SlackConfig               `yaml:"slack" bson:"slack" json:"slack" id:"slack"`
+	SpawnHostsPerUser         int                       `yaml:"spawn_hosts_per_user" bson:"spawn_hosts_per_user" json:"spawn_hosts_per_user"`
+	Splunk                    send.SplunkConnectionInfo `yaml:"splunk" bson:"splunk" json:"splunk"`
+	Triggers                  TriggerConfig             `yaml:"triggers" bson:"triggers" json:"triggers" id:"triggers"`
+	Ui                        UIConfig                  `yaml:"ui" bson:"ui" json:"ui" id:"ui"`
+	UnexpirableHostsPerUser   int                       `yaml:"unexpirable_hosts_per_user" bson:"unexpirable_hosts_per_user" json:"unexpirable_hosts_per_user"`
+	UnexpirableVolumesPerUser int                       `yaml:"unexpirable_volumes_per_user" bson:"unexpirable_volumes_per_user" json:"unexpirable_volumes_per_user"`
```

Comment:
Can we put this in a spawn host config block?

Reply:
done